src/components/NavBar/NavBar.scss

```diff
@@ -1,5 +1,159 @@
 @import url('https://fonts.googleapis.com/css?family=Raleway:600&display=swap');
 
+.navbar {
+  font-family: 'Raleway', sans-serif;
+  background-color: #1a1e23;
+  display: flex;
+  flex-direction: row;
+  font-weight: 600;
+
+  @media (max-width: 500px) {
+    display: flex;
+    flex-direction: column;
+  }
+
+  nav {
+    z-index: 103 !important;
+  }
+
+  .logo {
+    font-size: 30px;
+    display: flex;
+    justify-content: flex-end;
+    // padding-left: 8.4rem;
```

Comment:
if we're not using it, let's delete it.. I believe kids these days are calling it "yeet"